### DISCUSSION TITLE
windows平板精簡

### DISCUSSION THREAD

**Discussion by iranaldo** [WhatTheBlock/WindowsSimplify#22] [General]
我是使用windows平板,以往安裝過精簡版win10,發覺沒有平板功能及虛擬鍵盤,我的平板正是極需要精簡,因為是非常弱硬件,希望各位大大給建議

**Comment by WhatTheBlock** [comment]
你有試過老舊裝置分類的系統嗎
https://github.com/WhatTheBlock/WindowsSimplify/blob/master/10/old_device.md

**Reply by iranaldo** [reply]
好的，我先試試，感謝！
